app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/posts/EventPost.kt

```diff
@@ -173,12 +182,67 @@ fun EventPost(
               }
             }
 
+        if (currentUser == post.userId) {
+          IconButton(onClick = { showDeletePostDialog = true }) {
+            Icon(
+                imageVector = Icons.Default.Delete,
+                contentDescription = "Delete",
+                tint = MaterialTheme.colorScheme.onBackground)
+          }
+        }
+
         Spacer(modifier = Modifier.weight(1f))
         Text(
-            text = "${event.getDateString()}, ${event.getTimeString()}",
+            text = "${event.date}, ${event.time}",
```

Comment:
Why the change from using .getDateString() and .getTimeString() to this? Those functions ensure that the date and time as Strings are consistent everywhere in terms of format ("DD/MM/YYYY" or "Today" for date and "HH:MM" in 24h for time), whereas using the implicit toString function of LocalDate and LocalTime might not necessarily yield the same.

Reply:
<img width="248" alt="Screenshot 2024-05-26 at 23 03 14" src="https://github.com/SwEnt-Project-G18/SwEntApp/assets/74781400/c6367601-aa36-4ad3-be06-a36f4219cfc2">
Now the date/time looks like this (bottom right) rather than DD/MM/YYYY HH:MM

Reply:
Because ChatGPT trolled me when I asked him to clean the code a little bit

Reply:
Ahhh sacré GPT 🤖

Reply:
Should be fixed now